### PR TITLE
LCSD-2259 - update from new bc service card details

### PR DESCRIPTION
**Pull request by brianorwhatever**

### CODE REVIEW

File: cllc-public-app/Models.Extensions/Contact.cs [hunk 1]
```diff
@@ -161,8 +165,10 @@ public static void CopyHeaderValues(this ViewModels.Worker to, IHeaderDictionary
 
             // birthdate is YYYY-MM-DD
             string smgov_birthdate = headers["smgov_birthdate"];
+            smgov_birthdate = "1989-02-21";
             // Male / Female / Unknown. 
             string smgov_sex = headers["smgov_sex"];
+            smgov_sex = "Female";
```
Comment (GeorgeWalker) [inline cllc-public-app/Models.Extensions/Contact.cs]: This looks like it should be removed before we merge the PR

File: cllc-public-app/ClientApp/src/app/models/contact.model.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ export class Contact {
   lastname: string;
   emailaddress1: string;
   telephone1: string;
-  address1_street1: string;
+  address1_line1: string;
```
Comment (GeorgeWalker) [inline cllc-public-app/ClientApp/src/app/models/contact.model.ts]: Are changes to the Angular code required for this?

File: cllc-public-app/Models.Extensions/Contact.cs (hunk 1 above)
Comment (brianorwhatever) [reply cllc-public-app/Models.Extensions/Contact.cs]: good catch thanks

File: cllc-public-app/ClientApp/src/app/models/contact.model.ts (hunk 2 above)
Comment (brianorwhatever) [reply cllc-public-app/ClientApp/src/app/models/contact.model.ts]: no i did a global search. this was the only occurrence of `address1_street1` must have been a typo that didn't effect anything..

Comment (brianorwhatever) [reply cllc-public-app/ClientApp/src/app/models/contact.model.ts]: and when i say didn't effect anything i mean theres a chance something is relying on the incorrect behavior that was in place before..

File: cllc-public-app/Models.Extensions/Contact.cs (hunk 1 above)
Comment (brianorwhatever) [reply cllc-public-app/Models.Extensions/Contact.cs]: fixed

**Pull request merged**